server/jetty/src/main/java/io/deephaven/server/jetty/JettyBackedGrpcServer.java

```diff
@@ -97,9 +98,11 @@ public JettyBackedGrpcServer(
 
         // For the Web UI, cache everything in the static folder
         // https://create-react-app.dev/docs/production-build/#static-file-caching
```

Comment:
maybe update these comments while we're in here?

Reply:
I assumed the "static folder" meant something about the react build process; but in this context, should we instead say the "assets folder"? @mofojed

Reply:
Yea, we use Vite now instead of create-react-app, so we should update this comment. Though there doesn't seem to be a Vite equivalent page that is as helpful, closest I could find is: https://vitejs.dev/guide/dep-pre-bundling.html#browser-cache